### ISSUE TITLE
Data granularity

### ISSUE THREAD

**Issue by kevinykuo** [kasaai/pc-pricing-tutorial#59]
Some of the exposures are large, but they might actually be individual policies with many vehicles. Will have to investigate/ask.

**Comment by TylerGrantSmith** [comment]
I believe that every record is not an individual policy.  Each row is the unique combination of vehicle_category_code, region_code, vehicle_code, sex_code, age_code, and vehicle_year.

![image](https://user-images.githubusercontent.com/39497067/50199328-da1c6000-0315-11e9-91d8-85f786fc42e7.png)

**Comment by rafaelcosta1** [comment]
I agree. Also, each individual exposure can count to up to 0.5, as the database is for a 6-month period (even though the vast majority of auto policies in Brazil are annual.)

**Comment by RonRichman** [comment]
The aggregated rows shouldn't make a difference to the output of a GLM... What worries me a bit more is that some of the rows with the largest exposures seem to have a different premium rate than the single rows.
